operators/hpe-csi-operator/sources/hpecsidrivers.storage.hpe.com.crd.yaml

```diff
@@ -1187,10 +1297,9 @@ spec:
             - disableNodeConfiguration
             - disableNodeGetVolumeStats
             - imagePullPolicy
-            - iscsi
+            - images
```

Comment:
Just to understand, was that iscsi parameter wrongly added earlier?

Reply:
No, we have removed iscsi.chapUser and iscsi.chapPassword from the chart.